.github/workflows/test.yml

```
@@ -0,0 +1,50 @@
+    steps:
+    - uses: actions/checkout@v4
+    
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+        cache: 'yarn'
+    
+    - name: Install dependencies
+      run: yarn install --frozen-lockfile
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)